backend/app/services/inbox/auto_process_handler.py

```
@@ -112,6 +104,30 @@ async def on_message_created(self, event: QueueMessageCreatedEvent) -> None:
+                    )
+
+                    await inbox_direct_agent_handler.handle(
+                        event=event,
+                        auto_process=auto_process,
+                        message=message,
+                        work_queue=work_queue,
+                        db=db,
+                    )
+                    return
+
+                # Default: subscription mode – existing path below
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)